.github/workflows/build.yml

```
@@ -1,26 +1,27 @@
 name: Build Test
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push]
```

```suggestion
on: [push, pull_request]
```

Build tests should also cover PRs.